_includes/intro.html

```diff
@@ -1,15 +1,14 @@
 <section class="section">
 
   <div class="hero">
-    <h1>Learn to code or level up with RailsBridge!</h1>
+    <h1>Get started coding with RailsBridge!</h1>
 
-    <p>RailsBridge workshops are a free and fun way to get started or level up with Rails, Ruby, and other web technologies. Our events focus on increasing diversity in tech, so that people of all backgrounds can feel welcome and comfortable in our industry.</p>
+    <p>RailsBridge workshops are a <strong>free</strong> and fun way to get started or level up with Rails, Ruby, and other web technologies. Our events focus on increasing diversity in tech, so that people of all backgrounds can feel welcome and comfortable in our industry.</p>
   </div>
 
-  <p>The RailsBridge Workshops are <strong>free courses</strong> run by a volunteer team of <strong>experienced and enthusiastic developers</strong>. The courses cover an introduction to programming concepts, the available tools, Rails, Ruby, and HTML &amp; CSS.</p>
-
-  <p><span id="eligibility-criteria">The workshops are targeted at <strong>groups of people that are underrepresented in tech</strong></span>. The workshops teach <strong>practical skills</strong> useful to those investigating new career paths, looking to compliment their current skill set, or simply interested in gaining a better understanding of software.</p>
+  <p><span id="eligibility-criteria">RailsBridge Workshops are <strong>free courses</strong> targeted at <strong>groups of people that are underrepresented in tech</strong>.</span> The courses cover an introduction to programming concepts, the available tools, Rails, Ruby, HTML, CSS, and JavaScript.</p>
 
+  <p>A RailsBridge workshop is for you if you want to learn some practical skills and <strong>start learning to code</strong>, <strong>level up your skills</strong>, or <strong>connect with our volunteer team of experienced and enthusiastic developers</strong>. Read more at our <a href="/is-this-workshop-for-you.html">Is this workshop for you?</a>.</p>
```

Comment:
@patrickvine Thanks for the feedback! I've tweaked the wording here https://github.com/RailsBridge-CapeTown/rails-bridge-capetown.github.com/commit/48cd5726b3a2141df2d7979a0aab36ab607c2813 . I've split it into bullets so that it reads more human :).